cypress/integration/start_recipe.js

```diff
@@ -0,0 +1,13 @@
+describe('Starting new recipe test', () => {
+  it('Goes to recipe site', () => {
+    cy.visit('recipe');
+    cy.get('.chamber:nth-child(2)').click();
+    cy.get('button').contains('Load recipe', { timeout: 10000 }).click();
+    // wait for data otherwise test crashes
+    cy.intercept('/api/data').as('data');
+    cy.wait('@data');
+    cy.get('button').contains('Start brewing', { timeout: 10000 }).click();
+    cy.get('button').contains('Confirm', { timeout: 10000 }).click();
+    cy.get('[alt="arrow"]').should('be.visible');
+  });
```

Comment:
skus tam popridavat komentare pred tie commandy nech je jasnesie co sa deje, a skus tam mozno nechat medzeri medzi celkami, nech to je prehladnejsie, inak to je fajn cele